### ISSUE TITLE
Networth Report

### ISSUE THREAD

**Issue by spillaip** [moneymanagerex/moneymanagerex#6665]
# Description of the feature

Please describe your feature with details.
I understand MMEX comes with report generator/manager.  But, writing SQL is a complex expecation from the non-technical users.  How about adding few reports out-of-box:
1. Net worth Report (including, by Today, Month, Year along with Graphs) -- If this is already present, please let me know.
2. Cash flow report for this month, quarter, year and custom (I see a report already, but that is more a projection to future periods)
3. Investments Report (Capital Gains by Account, Capital Gains by Type, Holding Pie, Performance of Investments over different periods, list of investment transactions by accounts over a period. Summary of stocks is not intuitive and useful)
4. Taxes (by category current fiscal year and previous year, by patee current fiscal year and previous year)
5. Export of Categories, and Payees as CSV

Add screenshots, design images or other files which would help for
understanding the feature or for implementation.
Also add links when needed, for instance for implementation standards
or other relevant resources.

# Use cases

If not obvious, explain the use cases or problems to solve.

**Comment by ovari** [comment]
Is **Net Worth** shown on the Dashboard?

![image](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/9c1b4a0c-3cef-431f-ba21-6eb3ab3b4917)

**Comment by spillaip** [comment]
@ovari Yes, the dashboard shows all the accounts. But, networth is usually a report spanning multiple periods.

**Comment by ovari** [comment]
Navigation, Reports, Summary of Accounts, Monthly

![image](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/7f3c5807-338f-4e3d-80ec-71b1b4154cd2)

Is that the Net Worth spanning multiple periods?

**Comment by spillaip** [comment]
@ovari Account Balance by Month is different than Networth Report.  Usually Networth is a consolidation of all the account on a given day, like a balance sheet.

**Comment by ovari** [comment]
@spillaip could these suggestions, if implemented, help:

1. Add `Reports` option to the menu bar
![image](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/cbc468bc-e455-46b7-8400-4e74b534998b)
2. Move `General Reports Manager` to the `Reports` menu
3. Move `Transaction Report Filter` to the `Reports` menu
![image](https://github.com/moneymanagerex/moneymanagerex/assets/17465872/d147155b-6796-405d-8e93-e30d99f0090c)

Thank you

**Comment by spillaip** [comment]
@ovari I think this is a great step.  I am hoping these are canned reports.

**Comment by ovari** [comment]
They could be canned (pre-built) reports which can be customized.

**Comment by spillaip** [comment]
@ovari You are amazing.

**Comment by spillaip** [comment]
This is useful for many users, keep this on.

**Comment by nvskarunakar** [comment]
@ovari @spillaip Isn't Networth calculation shown in the dashboard/reports double counting "Share Account" balance which is included both in "Reconciled" and "Stock" values?

**Comment by spillaip** [comment]
@nvskarunakar I am unaware of the calculation behind the number, also @ovari planned canned report in future release. expecting for the same with lots of appreciation.

**Comment by spillaip** [comment]
Any update regarding the canned report?
